docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -117,7 +152,7 @@ $ oc wait --for condition=Ready --timeout=300s Nova/nova
 +
 [NOTE]
 The local Conductor services are started for each cell, while the superconductor runs in `cell0`.
-Note that `disable_compute_service_check_for_ffu` is mandatory for all imported Compute services until the external data plane is imported, and until Compute services are fast-forward upgraded. For more information, see xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane] and xref:performing-a-fast-forward-upgrade-on-compute-services_data-plane[Performing a fast-forward upgrade on Compute services].
+Note that `disable_compute_service_check_for_ffu` is mandatory for all imported Compute services until the external data plane is imported, and until Compute services are fast-forward upgraded. For more information, see xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane] and xref:performing-a-fast-forward-upgrade-on-compute-services_data-plane[Upgrading Compute services].
```

Comment:
++ thanks. This relates the separate discussion about batching adoption with adopting cell by cell. But that advanced feature is out of scope of this PR>